Misc/NEWS

```diff
@@ -10,6 +10,8 @@ Release date: XXXX-XX-XX
 Core and Builtins
 -----------------
 
+- Issue #26617: Fix crash when GC runs during weakref callbacks.
```

Comment:
If you ran blurb and entered data, and it didn't print an error, then it had an effect.  It creates a file in Misc/NEWS.d/next and runs "git add" on that file.

Were you inside this repo when you ran it?

If it's more convenient, you could copy an existing file in Misc/NEWS.d/next and manually change the filename to match the current time.  But we should figure out why blurb isn't working for you.